### PR TITLE
shauryam_post_added

### PR DESCRIPTION
**Pull request by shauryamjain**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: _pages/author-shauryam.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Shauryam Jain"
+layout: default
+permalink: "/author-shauryam-jain"
```
Comment (AvikantSrivastava) [inline _pages/author-shauryam.html:4]: The value of permalink should match with the file name

## Changes to make
1. Either make the permalink to `permalink: "/author-shauryam"`
2. Or change the filename to from ` _pages/author-shauryam.html` to ` _pages/author-shauryam-jain.html`

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
Everything else looks perfect, just make the changes that I have requested

File: _pages/author-shauryam.html (hunk 1 above)
Comment (KaushikiKrity) [reply _pages/author-shauryam.html:4]: I actually tried "/author-shauryam" but h link was not working in the local server but after making it to shauryam-jain it worked.

Comment (AvikantSrivastava) [reply _pages/author-shauryam.html:4]: Let me reproduced this issue on my local system, but probably the fix that I told should work

Comment (AvikantSrivastava) [reply _pages/author-shauryam.html:4]: Let me see it, I am working on Dockerizing the project

Comment (KaushikiKrity) [reply _pages/author-shauryam.html:4]: Any update on this or should we make a new PR.

**Pull request closed without merging**